src/box/iproto_constants.c

```diff
@@ -82,7 +82,7 @@ const unsigned char iproto_key_type[IPROTO_KEY_MAX] =
 	/* 0x22 */	MP_STR, /* IPROTO_FUNCTION_NAME */
 	/* 0x23 */	MP_STR, /* IPROTO_USER_NAME */
 	/* 0x24 */	MP_STR, /* IPROTO_INSTANCE_UUID */
-	/* 0x25 */	MP_STR, /* IPROTO_CLUSTER_UUID */
+	/* 0x25 */	MP_STR, /* IPROTO_REPLICASET_UUID */
```

Comment:
`IPROTO_CLUSTER_UUID` is documented [here](https://www.tarantool.io/en/doc/latest/dev_guide/internals/iproto/keys/). We should probably add a doc bot request to update the name there.

Reply:
Ok, added a request.